app/models/product.rb

```diff
@@ -49,7 +48,7 @@ def as_indexed_json(*)
     )
   end
 
-  has_many_attached :documents
+  has_many_attached :documents # Deprecated, to be migrated and removed
```

Comment:
We use `bin/rails notes` now and then to pick up code tasks like this...

```suggestion
  # TODO: Remove this once attatchments have been migrated
  has_many_attached :documents
```